draft-ietf-tls-esni.md

```diff
@@ -449,8 +440,8 @@ standard ClientHello, with the exception of the following rules:
    constructed as described below.
 1. The value of `ECHConfig.public_name` MUST be placed in the "server_name"
    extension.
-1. It MUST NOT include the "pre_shared_key" extension. [[OPEN ISSUE: The text
-   needs to be updated to reflect this rule.]]
+1. It MUST NOT include the "pre_shared_key" extension. (See
+   {{flow-resumption-oracle}}.)
 
 The client then constructs the ClientHelloInner message just as it does a
 standard ClientHello, with the exception of the following rules:
```

Comment:
(Noticed this while looking at the text for something else: the list below also has a reference to "non-ECK PSK" that should be dropped here.)

Reply:
I explicitly kept that, but I don't really feel strongly about keeping it. So I just removed it.

Reply:
Hehe, I mean this PR removes the definition of "ECH PSK" and "non-ECH PSK" so we probably shouldn't reference them. :-) I was hoping we'd no longer have to associate this extra boolean with each session.